acdcli/acd_fuse.py

```diff
@@ -505,6 +518,11 @@ def read(self, path, length, offset, fh) -> bytes:
         if node.size < offset + length:
             length = node.size - offset
 
+        """If we attempt to read something we just wrote, give it back"""
+        ret = self.wp.read(node.id, fh, offset, length)
+        if ret and len(ret) == length:
+            return ret
```

Comment:
This sounds like a good idea and is a good intro to fixing #383 :-)

Basically, all writes and xattrs and other ops could behave the same way - write into the buffer/cache, flush every once in a while into the cloud (or if buffer is about to overflow the 1GB). This way random writes wouldn't hammer amazon servers with all the intermediate changes.